docs/en/stack/ml/df-analytics/ml-trained-models.asciidoc

```diff
@@ -13,168 +13,146 @@ information about this process, see <<ml-supervised-workflow>> and
-examples below.
---
-
-The following Python snippet exports the trained model that you reference to a 
-JSON file:
+// NOTCONSOLE
 
-[source, py]
+[source, bash]
 --------------------------------------------------
-import json
-from elasticsearch import Elasticsearch
-from elasticsearch.client.ml import MlClient
-es_client = Elasticsearch('URL to your ES instance', http_auth=(username, password), use_ssl=True)
-ml_client = MlClient(es_client)
-result = ml_client.get_trained_models(model_id='your-model-id', decompress_definition=False, include=definition)
-compressed_df = result['trained_model_configs'][0]
-with open('model_filename.json', 'w') as handle:
-    handle.write(json.dumps(compressed_df))
+flights1-1607953694065
+flights0-1607953585123
+lang_ident_model_1
 --------------------------------------------------
 // NOTCONSOLE
 
+In our case, we would like to export the model with ID `flights1-1607953694065`.
+--
 
-The following Python snippet imports the model that stored in the JSON file to 
-a cluster:
-
-[source, py]
+2. Using `curl` from the command line, again use the
+{ref}/get-trained-models.html[Get Trained Model API] to export the entire model
+definition and save it to a JSON file.
++
+--
+[source, bash]
 --------------------------------------------------
-import json
-from elasticsearch import Elasticsearch
-from elasticsearch.client.ml import MlClient
-es_client = Elasticsearch(args.es, http_auth=(username, password), use_ssl=True, timeout=60)
-ml_client = MlClient(es_client)
-with open(filename, 'r') as handle:
-  compressed_model = json.loads(handle.read())
-for field in ('version', 'create_time', 'estimated_heap_memory_usage_bytes', 'estimated_operations', 'license_level', 'id','created_by'):
-  if field in compressed_model:
-    del compressed_model[field]
-ml_client.put_trained_model(model_id=model_id, body=compressed_model)
+curl -u elastic:changeme \
+  -X GET "http://localhost:9200/_ml/trained_models/flights1-1607953694065?exclude_generated=true&include=definition&decompress_definition=false" \
+    | jq '.trained_model_configs[0] | del(.model_id)' \
+    > flights1.json
 --------------------------------------------------
 // NOTCONSOLE
 
+A few observations:
 
-You can achieve the same by running a bash script. Populate the environment 
-variables:
-
-`ES_ADDRESS="https://username:password@elasticsearch-address"`
+* Exporting models requires using `curl` or a similar tool that can *stream*
+the model over HTTP into a file. Using the https://www.elastic.co/guide/en/
+kibana/current/console-kibana.html[Kibana Dev Tools console] will cause the
+browser to become unresponsive due to the size of exported models.
 
-`MODEL="my_model_name"`
+* Note the query parameters that are used during export. These are necessary to
+export the model in a way that it can later be imported again and used for
+inference.
 
+* We need to unnest the JSON object by one level to extract just the model
```

Comment:
This is another paragraph where ideally "we" can be replaced with "you".  For example:
```suggestion
* You must unnest the JSON object by one level to extract just the model
```